python/src/apx_agent/cli.py

```
@@ -5837,6 +5837,131 @@ def _kill_pid(pid: int, name: str) -> bool:
+
+    # --- request ---
+    endpoint = url.rstrip("/") + "/responses"
+    body = json.dumps({
+        "input": [{"role": "user", "content": question}],
+        "stream": use_stream,
+    }).encode()
+
+    click.echo(f"\n❓ {question}")
+    click.echo("─" * min(70, len(question) + 3))
+
+    req = urllib.request.Request(endpoint, data=body, headers=headers, method="POST")
```

Ruff (0.16.3)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)